yt_dlp/extractor/nhk.py

```diff
@@ -94,8 +94,21 @@ class NhkVodIE(NhkBaseIE):
     # Content available only for a limited period of time. Visit
     # https://www3.nhk.or.jp/nhkworld/en/ondemand/ for working samples.
     _TESTS = [{
+        'url': 'https://www3.nhk.or.jp/nhkworld/en/ondemand/video/2061601/',
+        'info_dict': {
+            'id': 'yd8322ch',
+            'ext': 'mp4',
+            'description': 'NHK WORLD-JAPAN presents a sumo highlights program for fans around the globe. Today the'
+                           ' show features all top-division bouts from May 14, Day 1 of the Grand Sumo Tournament in'
+                           ' Tokyo.',
```

Comment:
use md5 for long descriptions

Reply:
Converted to md5, also added missing values that were causing test failure.